components/Cards/SchoolCard/__tests__/BackSchoolCard.test.js

```diff
@@ -10,14 +11,40 @@ const locations = [
     city: 'San Francisco',
     zip: '94111',
   },
+  {
+    va_accepted: true,
+    address1: '825 Battery Street',
+    address2: '3rd Floor',
+    city: 'San Francisco',
+    zip: '94111',
```

Comment:
still need state here, and I'd use different data in the second object - you can make it up! have fun with it.

(same with front card tests)